src/common/base/NebulaKeyUtils.h

```diff
@@ -13,13 +13,21 @@ namespace nebula {
 
 /**
  * VertexKeyUtils:
- * partId(4) + vertexId(8) + tagId(4) + version(8)
+ * type(1) + partId(3) + vertexId(8) + tagId(4) + version(8)
  *
  * EdgeKeyUtils:
- * partId(4) + srcId(8) + edgeType(4) + edgeRank(8) + dstId(8) + version(8)
+ * type(1) + partId(3) + srcId(8) + edgeType(4) + edgeRank(8) + dstId(8) + version(8)
  *
  * */
 
+enum NebulaKeyType : uint32_t {
+    kData              = 0x00000001,
+    kIndex             = 0x00000002,
+    kUUID              = 0x00000003,
+    kSystemCommit      = 0x00000004,
```

Comment:
Just use one kSystem to represent key inside our system. 
And you could distinguish commit key and part key by suffix.